erpnext/accounts/doctype/sales_invoice/sales_invoice.json

```diff
@@ -1,1623 +1,1627 @@
 {
-   "allow_import": 1,
-   "autoname": "naming_series:",
-   "creation": "2013-05-24 19:29:05",
-   "doctype": "DocType",
-   "engine": "InnoDB",
-   "field_order": [
-    "customer_section",
-    "title",
-    "naming_series",
-    "customer",
```

Comment:
@jbienesdev why do you have indention issue with `Sales Invoice` json and the other 2 you dont have? Hard to see changes. Can you take a look?

Reply:
Sure @creamdory